From foo@baz.org Sun Jun  5 14:33:21 2005
Date: Mon, 20 Jun 2005 21:15:16 -0700
To: Greg KH <gregkh@suse.de>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: devfs: Last little devfs cleanups throughout the kernel tree.

Just removes a few unused #defines and fixes some comments due to
devfs now being gone.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


 Documentation/DocBook/kernel-api.tmpl             |    5 -----
 drivers/block/viodasd.c                           |    1 -
 drivers/cdrom/viocd.c                             |    1 -
 drivers/md/dm.c                                   |    2 +-
 drivers/media/dvb/ttusb-budget/dvb-ttusb-budget.c |   11 -----------
 drivers/serial/dz.c                               |    4 ----
 drivers/serial/serial_core.c                      |    2 +-
 drivers/usb/usb-skeleton.c                        |    4 ++--
 8 files changed, 4 insertions(+), 26 deletions(-)

---
--- gregkh-2.6.orig/drivers/block/viodasd.c
+++ gregkh-2.6/drivers/block/viodasd.c
@@ -59,7 +59,6 @@ MODULE_LICENSE("GPL");
  * numbers 0-255 we get a maximum of 32 disks.
  */
 #define VIOD_GENHD_NAME		"iseries/vd"
-#define VIOD_GENHD_DEVFS_NAME	"iseries/disc"
 
 #define VIOD_VERS		"1.64"
 
--- gregkh-2.6.orig/drivers/cdrom/viocd.c
+++ gregkh-2.6/drivers/cdrom/viocd.c
@@ -51,7 +51,6 @@
 #include <asm/iseries/vio.h>
 
 #define VIOCD_DEVICE			"iseries/vcd"
-#define VIOCD_DEVICE_DEVFS		"iseries/vcd"
 
 #define VIOCD_VERS "1.06"
 
--- gregkh-2.6.orig/drivers/media/dvb/ttusb-budget/dvb-ttusb-budget.c
+++ gregkh-2.6/drivers/media/dvb/ttusb-budget/dvb-ttusb-budget.c
@@ -125,10 +125,6 @@ struct ttusb {
 
 	int revision;
 
-#if 0
-	devfs_handle_t stc_devfs_handle;
-#endif
-
 	struct dvb_frontend* fe;
 };
 
@@ -1575,13 +1571,6 @@ static int ttusb_probe(struct usb_interf
 		return -ENODEV;
 	}
 
-#if 0
-	ttusb->stc_devfs_handle =
-	    devfs_register(ttusb->adapter->devfs_handle, TTUSB_BUDGET_NAME,
-			   DEVFS_FL_DEFAULT, 0, 192,
-			   S_IFCHR | S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP
-			   | S_IROTH | S_IWOTH, &stc_fops, ttusb);
-#endif
 	usb_set_intfdata(intf, (void *) ttusb);
 
 	frontend_init(ttusb);
--- gregkh-2.6.orig/drivers/serial/dz.c
+++ gregkh-2.6/drivers/serial/dz.c
@@ -771,11 +771,7 @@ void __init dz_serial_console_init(void)
 static struct uart_driver dz_reg = {
 	.owner			= THIS_MODULE,
 	.driver_name		= "serial",
-#ifdef CONFIG_DEVFS
-	.dev_name		= "tts/%d",
-#else
 	.dev_name		= "ttyS%d",
-#endif
 	.major			= TTY_MAJOR,
 	.minor			= 64,
 	.nr			= DZ_NB_PORT,
--- gregkh-2.6.orig/drivers/md/dm.c
+++ gregkh-2.6/drivers/md/dm.c
@@ -151,7 +151,7 @@ static void local_exit(void)
 	bioset_free(dm_set);
 
 	if (unregister_blkdev(_major, _name) < 0)
-		DMERR("devfs_unregister_blkdev failed");
+		DMERR("unregister_blkdev failed");
 
 	_major = 0;
 
--- gregkh-2.6.orig/drivers/serial/serial_core.c
+++ gregkh-2.6/drivers/serial/serial_core.c
@@ -2280,7 +2280,7 @@ int uart_remove_one_port(struct uart_dri
 	down(&port_sem);
 
 	/*
-	 * Remove the devices from devfs
+	 * Remove the devices from sysfs
 	 */
 	tty_unregister_device(drv->tty_driver, port->line);
 
--- gregkh-2.6.orig/drivers/usb/usb-skeleton.c
+++ gregkh-2.6/drivers/usb/usb-skeleton.c
@@ -232,9 +232,9 @@ static struct file_operations skel_fops 
 	.release =	skel_release,
 };
 
-/* 
+/*
  * usb class driver info in order to get a minor number from the usb core,
- * and to have the device registered with devfs and the driver core
+ * and to have the device registered with sysfs.
  */
 static struct usb_class_driver skel_class = {
 	.name =		"skel%d",
--- gregkh-2.6.orig/Documentation/DocBook/kernel-api.tmpl
+++ gregkh-2.6/Documentation/DocBook/kernel-api.tmpl
@@ -309,11 +309,6 @@ X!Earch/i386/kernel/mca.c
      </sect1>
   </chapter>
 
-  <chapter id="devfs">
-     <title>The Device File System</title>
-!Efs/devfs/base.c
-  </chapter>
-
   <chapter id="sysfs">
      <title>The Filesystem for Exporting Kernel Objects</title>
 !Efs/sysfs/file.c
